docs/source/algorithms/CalculatePlaczekSelfScattering-v1.rst

```diff
@@ -0,0 +1,88 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm takes an incident spectrum function and it's derivative and
+calculates the time-of-flight Placzek scattering correction from the workspaces
+material sample and detector info. [1]_ [2]_ [3]_ For obtaining the incident spectrum
+from a measurement (ie beam monitors or calibrant sample), the
+:ref:FitIncidentSpectrum <algm-FitIncidentSpectrum> can provide the necessary inputs.
+
+Usage
+-----
+
+**Example: calculate Placzek self scattering correction using a sample detector setup** [4]_
+
+.. code:: python
+
+    from mantid.simpleapi import *
+    import matplotlib.pyplot as plt
+    import numpy as np
+
+    # Create the workspace to hold the already corrected incident spectrum
+    incident_wksp_name = 'incident_spectrum_wksp'
+    binning_incident = "%s,%s,%s" % (0.1, 0.02, 5.0)
```

Comment:
Does doing this string formatting give any benefit? I feel it probably confuses users more than it helps.